<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IqtFit</class>
 <widget class="QWidget" name="IqtFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitDataView" name="fitDataView" native="true"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="loPlotArea" stretch="1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MantidQt::MantidWidgets::IndirectFitPropertyBrowser" name="fitPropertyBrowser">
           <widget class="QWidget" name="dockWidgetContents"/>
          </widget>
         </item>
         <item>
          <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitPlotView" name="pvFitPlotView" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView" name="svSpectrumView" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="gbOutput">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbPlotType">
            <property name="text">
             <string>Plot Output: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbPlotType">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Background</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Intensity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tau</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Beta</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPlot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Plot Result</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save Result</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView</class>
   <extends>QWidget</extends>
   <header>IndirectSpectrumSelectionView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::IndirectFitPropertyBrowser</class>
   <extends>QDockWidget</extends>
   <header>MantidQtWidgets/Common/IndirectFitPropertyBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitPlotView</class>
   <extends>QWidget</extends>
   <header>IndirectFitPlotView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitDataView</class>
   <extends>QWidget</extends>
   <header>IndirectFitDataView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
